#158423707 Add drivers expanded ride view

artorious opened the pull request:
#### What does this PR do?

* Adds a drivers view of an expanded ride offer's details.

* A driver shuold be able to view all the details associated with that particular ride offer, with an option to accept or reject requests from passengers.

* A driver should also be able to close down the ride offer at will (E.g When he is at full capacity)

#### Description of Task to be completed?

* Building (without functionality), the UI elements with HTML&CSS. 

#### How the app should be tested
On a webbrowser:

* [Download](https://github.com/artorious/Ride-My-Way/archive/ft-drivers-expanded-ride-#158423707.zip) the repo
* Extract and navigate into the `Ride-My-Way` folder
* in the `UI` folder, open `expanded_ride_driver.html` on a browser

On a  terminal (Linux):

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone https://github.com/artorious/Ride-My-Way.git`

* Run `git checkout ft-drivers-expanded-ride-#158423707`

* Run `cd Ride-My-Way/UI/`

* Open `expanded_ride_driver.html` on a browser

#### Any background context you want to provide?
The UI has no functionality. 
Developed using HTML and CSS

#### What are the relevant pivotal tracker stories?
[#158423707](https://www.pivotaltracker.com/story/show/158423707)

#### Screenshots 
![expridedriver](https://user-images.githubusercontent.com/15730750/41740419-9f16f388-75a0-11e8-8e32-40744c5970a3.png)

#### Questions:

artorious committed:
[Feature #158423707] Add expanded ride details

Adds drivers view of a singe ride's details with minimal page styling

artorious committed:
[Feature #158423707] Style expanded ride view

Add page styling to drivers view of ride offer details.

[finishes #158423707]

artorious committed:
[Feature #158423707] Style expanded ride view

Add page styling to drivers view of ride offer details.

[finishes #158423707]

artorious committed:
Merge branch 'ft-drivers-expanded-ride-#158423707' of https://github.com/artorious/Ride-My-Way into ft-drivers-expanded-ride-#158423707

ghost commented:
The code is okay and design is good

xcixor commented:
Great idea to have the status as a dropdown and if I may suggest white text color could be better on the black backgrounds

meshack-mbuvi commented:
Same observations as @xcixo has made. Also, what would be the effect if you reduce the width of the labels for ride details? Try and see whether it becomes more user friendly

JoshuaOndieki commented on UI/expanded_ride_driver.html:
```diff
@@ -0,0 +1,125 @@
+    </tr>
+    <tr>
+      <th>Final Destination</th>
+      <td>petergriffin@email.com</td>
```
I assume your final destination is not an email address

artorious replied:
Thanks for that :+1:

JoshuaOndieki commented on UI/static/main.css:
```diff
@@ -120,4 +149,9 @@ footer a {
     }
+}
+
+/* Handle wide tables */
```
:sparkles: Great work with handling the tables! :+1: :sparkles:

artorious commented:
Hi @xcixor and @meshack-mbuvi 
I've increased the font size on the select buttons and changed color to white

@meshack-mbuvi please add color on `user-friendly`

![3](https://user-images.githubusercontent.com/15730750/41666849-2ab8693c-74b4-11e8-80b1-dc58f0f6d28e.png)
![4](https://user-images.githubusercontent.com/15730750/41666854-2dcc65b0-74b4-11e8-9ff5-fd2e2569f6c6.png)

artorious committed:
[Bug #158423707] Improve style on buttons.

Increase font size on buttons and change color to white for readabilty

meshack-mbuvi commented:
It looks good. Though you can consider reducing the amount of black color.You can have black color occupying atmost 30% of your design

LWanjiru commented:
@artorious Consider improving the look and feel of this page. There is a lot going on in terms especially of the background colours. Is it possible to have a more consistent feel? 

Curious question: what is the inspiration behind where the `status` drop-down appears?

artorious committed:
[Feature #158423707] Improve page aesthetics

Reduces the amount of black backround that was distracting from the
content. Also moved the ride-status selector to a more appropriate
position

artorious commented:
@meshack-mbuvi @rethes @xcixor @JoshuaOndieki @LWanjiru Thank you for your input and direction. I have toned down the `black-heavy` look of the page, I now realize that it was distracting from the content. I've also moved the `Ride status` selector to a more practical and suited position.

![expridedriver](https://user-images.githubusercontent.com/15730750/41741034-99e42f32-75a2-11e8-949d-62b24f71dff2.png)

artorious committed:
Merge branch 'UI-dev-features' into ft-drivers-expanded-ride-#158423707
